test/cbmc/stubs/core_sntp_stubs.c

```diff
@@ -77,11 +78,20 @@ int32_t NetworkInterfaceReceiveStub( NetworkContext_t * pNetworkContext,
     if( tries < ( MAX_NETWORK_RECV_TRIES - 1 ) )
     {
         tries++;
+
+        if( bytesOrError > 0 )
+        {
+            read += bytesOrError;
```

Comment:
Can you add a comment about the purpose of this logic as accumulating all the bytes read across multiple tries before the last try?